Program.fs

```diff
@@ -54,21 +54,18 @@ module MainModule =
     let apiDomain = Env.varDefault "ARTIST_TALLY_API_DOMAIN" "localhost:3000"
     let isLive = (Env.varDefault "ARTIST_TALLY_ENV" "development") = "production"
 
-    let buildMessage<'bodyType> method url authHeader (body: 'bodyType option) = 
-        let uri = Uri url
-        let message = new Http.HttpRequestMessage ()
-        message.Method <- method
-        message.RequestUri <- uri
+    let buildMessage method url authHeader body = 
+        let message = new Http.HttpRequestMessage (Method = method, RequestUri = Uri url)
```

Comment:
I was wondering if this was possible, but I just assumed it wasn't after realizing there were no initializers in the language. Happy to be wrong in this case!